dpc-sdp/sdp-platform/src/composer/Plugin.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace SdpPlatform\composer;
+
+use Composer\Composer;
+use Composer\EventDispatcher\EventSubscriberInterface;
+use Composer\Factory;
+use Composer\IO\IOInterface;
+use Composer\Plugin\PluginInterface;
+use Composer\Script\Event;
+use Composer\Script\ScriptEvents;
+
+class Plugin implements PluginInterface, EventSubscriberInterface {
+
+  /**
+   * @var \Composer\Composer
+   */
+  protected $composer;
+
+  /**
+   * @var \Composer\IO\IOInterface
+   */
+  protected $io;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function activate(Composer $composer, IOInterface $io) {
+    $this->composer = $composer;
+    $this->io = $io;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function getSubscribedEvents() {
+    return array(
+      ScriptEvents::POST_INSTALL_CMD => 'addPlatformFiles',
+      ScriptEvents::POST_UPDATE_CMD => 'addPlatformFiles',
+    );
+  }
+
+  /**
+   * Add/update platform files.
+   *
+   * @param \Composer\Script\Event $event
+   */
+  public function addPlatformFiles(Event $event) {
```

Comment:
This is obviously very brute force for the minute... it might make sense to still use ansible for some of this. Thoughts on how possible/easy to follow this would be?